inst/rmd/qc_report.rmd

```diff
@@ -204,17 +207,39 @@ knitr::kable(processing_info, align = 'r') |>
 
 ```{r}
 basic_statistics <- tibble::tibble(
-  "Filtering method"                   = metadata(filtered_sce)$filtering_method, 
-  "Number of cells post filtering"     = format(ncol(filtered_sce), big.mark = ","),
-  "Percent of reads in cells"          = paste0(round((sum(filtered_sce$sum)/sum(unfiltered_sce$sum))*100,2),"%"),
-  "Median UMI count per cell"          = format(median(filtered_sce$sum), big.mark = ","),
-  "Median genes detected per cell"     = format(median(filtered_sce$detected), big.mark = ","),
-  "Median percent reads mitochondrial" = paste0(round(median(filtered_sce$subsets_mito_percent), 2), "%")
-  ) |>
-  mutate(across(.fns = ~ifelse(.x == "NULL", "N/A", .x))) |>
-  t()
+  "Method used to filter empty droplets"          = metadata(filtered_sce)$filtering_method, 
+  "Number of cells post filtering empty droplets" = format(ncol(filtered_sce), big.mark = ","),
+  "Percent of reads in cells"                     = paste0(round((sum(filtered_sce$sum)/sum(unfiltered_sce$sum))*100,2),"%"),
+  "Median UMI count per cell"                     = format(median(filtered_sce$sum), big.mark = ","),
+  "Median genes detected per cell"                = format(median(filtered_sce$detected), big.mark = ","),
+  "Median percent reads mitochondrial"            = paste0(round(median(filtered_sce$subsets_mito_percent), 2), "%")
+  )
+
+# if processed sce exists add filtering and normalization table
+if(has_processed){
+  processed_meta <- metadata(processed_sce)
+  
+  basic_statistics <- basic_statistics |>
+    mutate(
+      "Method used to filter low quality cells" = format(processed_meta$ccdl_filter_method),
+      "Cells after filtering low quality cells" = format(dim(processed_sce)[2], big.mark = ',', scientific = FALSE),
+      "Normalization method"                    = format(processed_meta$normalization),
+      "Minimum genes per cell cutoff"           = format(processed_meta$min_gene_cutoff)
```

Comment:
I noticed in the report that this was NA, but this should usually be set, correct? Or are we not using a minimum even with miQC?

Reply:
this is usually set but we haven't added it to the metadata yet, so we need to add it to the metadata of the SCE object for it to print the value here. That needs to happen in `scpca-nf` though and not here.